library/firewall_lib.py

```diff
@@ -683,7 +715,7 @@ def exception_handler(exception_message):
                 changed = True
```

Comment:
For runtime, this code uses `fw.addInterface(zone, interface)`
however, the a.p.firewalld code uses `fw.changeZoneOfInterface(zone, interface)` instead - https://github.com/ansible-collections/ansible.posix/blob/main/plugins/modules/firewalld.py#L459

Which one is correct? @erig0 or @t-woerner ?